apps/explorer/src/components/orders/DetailsTable/index.tsx

```diff
@@ -267,12 +290,19 @@ export function DetailsTable(props: Props): JSX.Element | null {
                     <RowWithCopyButton
                       textToCopy={txHash}
                       onCopy={(): void => onCopy('settlementTx')}
-                      contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
+                      contentsToDisplay={<LinkWithPrefixNetwork to={getExplorerLink(chainId, txHash, ExplorerDataType.TRANSACTION)} target='_blank'>{txHash}↗</LinkWithPrefixNetwork>}
                     />
-                    <LinkButton to={`/tx/${txHash}/?${TAB_QUERY_PARAM_KEY}=graph`}>
-                      <FontAwesomeIcon icon={faProjectDiagram} />
-                      View batch graph
-                    </LinkButton>
+                    <Wrapper gap={false}>
+                      <LinkButton to={`/tx/${txHash}/?${TAB_QUERY_PARAM_KEY}`}>
```

Comment:
No need for the query param

```suggestion
                      <LinkButton to={`/tx/${txHash}`}>
```